src/azul/health.py

```diff
@@ -258,14 +245,24 @@ def api_endpoints(self):
         """
         Indicates whether important service API endpoints are operational.
         """
-        endpoints = [
+        # Select any single entity index endpoint from the ones available
+        random.seed(a=1695859751)
+        endpoint = random.choice([
             furl(path=('index', entity_type), args={'size': '1'})
             for entity_type in self.entity_types
-        ]
-        with ThreadPoolExecutor(len(endpoints)) as tpe:
-            status = dict(tpe.map(self._api_endpoint, endpoints))
-        status['up'] = all(v['up'] for v in status.values())
-        return status
+        ])
+        endpoint = str(config.service_endpoint.join(endpoint))
+        log.debug('_request(HEAD, %s) …', endpoint)
+        start = time.time()
+        response = requests.head(endpoint)
+        log.debug('_request(…) -> %.3fs %r',
+                  time.time() - start, response.status_code)
```

Comment:
```suggestion
        log.debug('_request(…) -> %.3fs %r',
                  time.time() - start, response.status_code)
```
If this log format was discussed or requested then it's ok, however personally something more readable would be my preference, e.g.

DEBUG: Requesting https://www.example.com/index/files 
DEBUG: Received a 200 response in 0.405 seconds

Reply:
Yeah, this is more or less how we log a requests' execution time. Checkout [src/azul/terra.py#L368](https://github.com/DataBiosphere/azul/blob/042165077853fe341cd0d4e96677ad9140bd4170/src/azul/terra.py#L343-#L368).